src/logic/proposals/selectors.ts

```diff
@@ -151,12 +151,11 @@ export const genesisEpochOverSelector = (isGenesisEpoch: boolean | undefined) =>
 };
 
 export const canVoteSelector = (proposal: Proposal) => {
-  return (
-    proposal.open &&
-    proposal.userVotingPowerAt.gt(0) &&
-    !proposal.delegateAt &&
-    VOTER_STATES[proposal.voterState] === 'Unvoted'
-  );
+  return {
+    isOpen: proposal.open,
+    hasEnoughVotingPower: proposal.userVotingPowerAt.gt(0),
+    isNotDelegated: !proposal.delegateAt,
```

Comment:
Shame on me for adding the `Unvoted` check... Not sure why did I assume it's needed.